relay-server/src/actors/project.rs

```diff
@@ -108,6 +108,9 @@ pub struct ProjectConfig {
     /// Configuration for sampling traces, if not present there will be no sampling.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub dynamic_sampling: Option<SamplingConfig>,
+    /// Configuration for measurements.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub measurements: Option<MeasurementsConfig>,
```

Comment:
nit: I'd be more explicit by mentioning somehow it's about transactions
```suggestion
    /// Configuration for transaction measurements.
    #[serde(skip_serializing_if = "Option::is_none")]
    pub transaction_measurements: Option<MeasurementsConfig>,
```
(same for the implementation of `Default` and `LimitedProjectConfig` below.)

Reply:
I see your point, but I would rather keep this consistent with `breakdowns_v2` just below, which is also transaction-specific but does not mention that in the name.